### ISSUE TITLE
Newsfeed curated by top wallets

### ISSUE THREAD

**Issue by IkigaiLabsETH** [IkigaiLabsETH/ikigailabs.xyz#112]

**Comment by royscheeren** [comment]
I really don't know what to build with this information

**Comment by IkigaiLabsETH** [comment]
I wanna create an array of wallets from collectors and see their activity feed with https://docs.reservoir.tools/reference/getusersactivityv4

**Comment by royscheeren** [comment]
Ok, and you're happy to update the wallets in the config file?

**Comment by IkigaiLabsETH** [comment]
Yes that's perfectly fine to do it there, I found https://artacle.io/leaderboard?sortBy=aVal where I get to curated wallets that I wanna track
